data-structure/README.md

```diff
@@ -290,7 +309,7 @@ An example of a `Work`, and its physical and digital `Manifestation`s:
     "@type": "CreativeWork",
     "@id": "<URI pointing to this object>",
     "name": "Lord of the Rings",
-    "author": "<URI pointing to a Person or Organization object>"
+    "creator": "<URI pointing to a Person or Organization object>"
```

Comment:
Good point.

There can probably be even a case where an `author` and a `creator` is necessary.